src/django/api/views/__init__.py

```diff
@@ -23,7 +23,7 @@
     ContributorFacilityListSortedViewSet
 )
 from .contributor.get_contributor import get_contributor
-from .contributor.contributor_webhook_view_set import ContributorWebhookViewSet
+# from .contributor.contributor_webhook_view_set import ContributorWebhookViewSet
```

Comment:
Why did you comment it out? If it's a typo, please delete this line.

Reply:
removed